etc/requirements.txt

```diff
@@ -2,4 +2,5 @@
 Click
 GitPython
 PyGithub
-jira
\ No newline at end of file
+jira
+cryptography
```

Comment:
Recommend version-pinning this to a known-good version, especially as this package is "notable" for breaking changes. (all packages here should be pinned, as well as their transitive dependencies, probably). Refer to: "`pip freeze`"